aicsdeformation/loaders/loader_abc.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+import numpy as np
+
+
+class LoaderABC(ABC):
```

Comment:
I thought loaders were just to get the frame data. Take object(s) and return list of numpy 2d arrays? These abstract methods do neither to my understanding.